### PR TITLE
The switch to py27 was done a long time ago

### PR DESCRIPTION
**Pull request by magopian**

### CODE REVIEW

File: scripts/monitors/check_wsgi.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.6
+#!/usr/bin/python2.7
```
Comment (tofumatt) [inline scripts/monitors/check_wsgi.py]: Would this be more future-proof as `/usr/bin/env python`?

File: docs/topics/install-olympia/installation.rst [hunk 2]
```diff
@@ -143,13 +143,12 @@ To get back into the olympia environment later, type::
...
 
-.. note:: Olympia requires at least Python 2.6.1, and at most Python 2.7,
-          production is using Python 2.6.6.
+.. note:: Olympia requires Python 2.7.
 
 .. note:: If you want to use a different Python binary, pass the name (if it is
           on your path) or the full path to mkvirtualenv with ``--python``::
 
-            mkvirtualenv --python=/usr/local/bin/python2.6 olympia
+            mkvirtualenv --python=/usr/local/bin/python2.7 olympia
```
Comment (kumar303) [inline docs/topics/install-olympia/installation.rst]: this could just be `--python=python2.7` since it should always be on the path

Comment (magopian) [inline docs/topics/install-olympia/installation.rst]: Well, the point of this note is to explain how to use an alternative python binary (maybe not on the path) for the virtualenv

**Comment by kumar303** [comment]
r+

File: scripts/monitors/check_wsgi.py (hunk 1 above)
Comment (magopian) [inline scripts/monitors/check_wsgi.py]: @jasonthomas is `/usr/bin/env python` the expected py2.7 interpreter on production?

To be honest I'm not even sure this script is used at all

Comment (jasonthomas) [inline scripts/monitors/check_wsgi.py]: /usr/bin/env python will get us the correct python version.

We aren't using this script anywhere today for monitoring so I think we can just delete it.

Comment (magopian) [inline scripts/monitors/check_wsgi.py]: Ah, will do that instead then, thanks ;)

File: scripts/monitors/check_wsgi.py [hunk 3]
```diff
@@ -1,43 +0,0 @@
-#!/usr/bin/python2.6
```
Comment (magopian) [inline scripts/monitors/check_wsgi.py]: Instead of modifying this shebang, took the opportunity to entirely remove the file which isn't used anymore.

**Pull request merged**